src/database/db_build.js

```diff
@@ -2,7 +2,7 @@ const connection = require('./db_connection.js');
 
 const runDatabaseBuild = (db) => {
   connection().then((dbs) => {
-    const database = dbs[db];
+    const database = dbs.production;
```

Comment:
@ahmedisam99 You could keep `dbs[db]` and have `production` be the default value for the db argument:
`const runDatabaseBuild = (db = 'production') => {`
To keep this function more general-use.